docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -33,7 +33,7 @@ These other tools look for the user config file under the `~/.config/NuGet` fold
 
 If the tooling you use involves both locations, consider consolidating them by following these steps to allow you to work with only one user-level config file:  
 1. Check the contents of the two user-level config files and keep the one you want under `~/.nuget/NuGet` folder. 
-2. Set symbolic link from `~/.nuget/NuGet` to `~/.config/Nuget`. E.g. Run bash command: `ln -s ~/.nuget/NuGet ~/.config/Nuget`.
+2. Set symbolic link from `~/.nuget/NuGet` to `~/.config/NuGet`. E.g. Run bash command: `ln -s ~/.nuget/NuGet ~/.config/NuGet`.
```

Comment:
Thank you for your contribution.
Did you test this with both MAC and Linux?

Reply:
Please note there is another similar open discussion on this topic. Have you looked into it?
https://github.com/NuGet/docs.microsoft.com-nuget/pull/2968/files#r1052464087

Reply:
Based on our code, I don't think we've ever used `Nuget`. 

https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Common/PathUtil/NuGetEnvironment.cs